SECURITY.md

```diff
@@ -6,25 +6,41 @@ This is the security notice for all GDS (Government Digital Service) repositorie
 
 ## Reporting a Vulnerability
 
-GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.
+GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it. Quick links can be found at our [security.txt](https://vdp.cabinetoffice.gov.uk/.well-known/security.txt).
```

Comment:
Suggest being a bit clearer on what we mean by "quick links", for example, by saying something like...

`This notice provides details for how you can let us know about vulnerabilities, or alternatively you can view our [security.txt](https://vdp.cabinetoffice.gov.uk/.well-known/security.txt) file which contains quick links to contact us.`